pkg/rbac/validate_test.go

```diff
@@ -287,3 +289,21 @@ func TestCan(t *testing.T) {
 		})
 	}
 }
+
+func TestObjectMatch(t *testing.T) {
+	t.Parallel()
+	assert.True(t, objectMatch("", "*"))
+	assert.True(t, objectMatch("namespace-1/object-1", "*"))
```

Comment:
These are the cases which broke when we used `globMatch` instead. Using `keyMatch() || globMatch` also did not work, since it wrongly allowed certain cases..